py/desimodel/test/test_install.py

```diff
@@ -19,27 +20,54 @@ class TestInstall(unittest.TestCase):
     """Test desimodel.install.
     """
 
+    def setUp(self):
+        self.orig_version = desimodel.__version__
+
+    def tearDown(self):
+        desimodel.__version__ = self.orig_version
+
     def test_default_install_dir(self):
         """Test setting default install directory.
         """
         d = os.path.dirname
         d1 = default_install_dir()
-        d2 = d(d(d(d(d(d(__file__))))))
+        d2 = d(d(__file__))
         self.assertEqual(os.path.abspath(d1), os.path.abspath(d2))
 
     def test_svn_export(self):
         """Test svn export command.
         """
         base_url = "https://desi.lbl.gov/svn/code/desimodel/{0}/data"
+
+        desimodel.__version__ = '1.2.3.dev456'
+        cmd = svn_export()
+        self.assertEqual(cmd[2], base_url.format('trunk'))
+
+        desimodel.__version__ = '7.8.9'
+        cmd = svn_export()
+        self.assertEqual(cmd[2], base_url.format('tags/7.8.9'))
+
+        desimodel.__version__ = '7.10'
+        cmd = svn_export()
+        self.assertEqual(cmd[2], base_url.format('tags/7.10'))
+
+        desimodel.__version__ = 'blatfoo'
         cmd = svn_export()
         self.assertEqual(cmd[2], base_url.format('trunk'))
+
+        desimodel.__version__ = self.orig_version
         cmd = svn_export('trunk')
         self.assertEqual(cmd[2], base_url.format('trunk'))
+
         cmd = svn_export('branches/v3')
         self.assertEqual(cmd[2], base_url.format('branches/v3'))
+
         cmd = svn_export('1.2.3')
         self.assertEqual(cmd[2], base_url.format('tags/1.2.3'))
 
+        cmd = svn_export('1.2')
+        self.assertEqual(cmd[2], base_url.format('tags/1.2'))
+
```

Comment:
These are all good tests, but we definitely need to test the scenario where we install a tag *or* main/trunk, but we need to install a test branch.

For example, I need to test version 0.19.2 but I don't want to download the full data set, just `branches/test-0.19`.

Reply:
Good point.  I added a test for that (not yet pushed).

For the record, this does require specifying "branches" in the name like you have above, e.g.
```
install_desimodel_data --desimodel-version branches/test-0.19
```
since the code check is that "branches/" means it's not a tag, not "test-" means it's not a tag.  If you run
```
install_desimodel_data --desimodel-version test-0.19
```
it will try to find `tags/test-0.19` instead of `branches/test-0.19`.  I think that is ok, but wanted to be explicit about it here.